### PR TITLE
new sru docs for LP: #1819067

### PR DESCRIPTION
**Pull request by OddBloke**

### CODE REVIEW

File: manual/azure-sru-18.5-44-g7c07af28-0ubuntu1.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Manually deploy xenial and artful on Azure using Azure CLI client.
```
Comment (raharper) [inline manual/azure-sru-18.5-44-g7c07af28-0ubuntu1.txt]: This has Artful, header should be Xenial, Bionic, Cosmic.

File: manual/nocloud-kvm-18.5-44-g7c07af28-0ubuntu1.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+###  Verification script
+
+The following is console output of  integration test run of xenial proposed citests run by blackboxsw (chad.smith)
+
+=== Xenial ===
+tox -e citest -- run --os-name=xenial --platform=nocloud-kvm --repo="deb http://archive.ubuntu.com/ubuntu/ xenial-proposed main" --preserve-data --data-dir=../results --verbose;
+==== Artful ====
```
Comment (raharper) [inline manual/nocloud-kvm-18.5-44-g7c07af28-0ubuntu1.txt]: s/Artful/Bionic & Cosmic

File: manual/nocloud-lxd-18.5-44-g7c07af28-0ubuntu1.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+=== Xenial automated integration tests nocloud-lxd from https://jenkins.ubuntu.com/server/view/cloud-init/job/cloud-init-integration-lxd-x/465/consoleFull
+
+=== Artful automated nocloud-kvm integration test logs from https://jenkins.ubuntu.com/server/view/cloud-init/job/cloud-init-integration-lxd-a/275/consoleFull
```
Comment (raharper) [inline manual/nocloud-lxd-18.5-44-g7c07af28-0ubuntu1.txt]: Artful here again

**Comment by OddBloke** [comment]
I believe I've removed the artful references from both the templates and the new manual files generated from them.

**Pull request merged**